### ISSUE TITLE
2.4.2　Variable简介代码更新

### ISSUE THREAD

**Issue by JimXiongGM** [xiaobaoonline/pytorch-in-action#4]
应该更新为：
>>>from torch import tensor
>>>x = torch.rand(4)
>>>x = x.clone().detach().requires_grad_(True)
>>>y= x * 3
>>>grad_variables = torch.FloatTensor([1,2,3,4])
>>>y.backward(grad_variables)
>>>x.grad
tensor([  3.,   6.,   9.,  12.])

**Comment by jiangyangbo** [comment]
import torch
from torch import tensor
x = torch.rand(4, requires_grad=True)
y= x * 3
grad_variables = torch.FloatTensor([1,2,3,4])
y.backward(grad_variables)
print(x.grad)

**Comment by jiangyangbo** [comment]
谢谢，原来代码是0.3版本的。

from torch import tensor
import torch
x = torch.rand(4, requires_grad=True)
y= x * 3
grad_variables = torch.FloatTensor([1,2,3,4])
y.backward(grad_variables)
print(x.grad)
